tests/metrics/test_pcr_metrics.py

```diff
@@ -18,18 +18,17 @@ def test_pcr_batch(adata):
     assert 0 <= score < 1e-6
 
 
-def test_pcr_batch_precomputed(adata, adata_pca):
-    adata = adata_pca(adata)
-    score = scIB.me.pcr_comparison(adata, adata, covariate='batch', scale=True)
+def test_pcr_batch_precomputed(adata_pca):
+    score = scIB.me.pcr_comparison(adata_pca, adata_pca, covariate='batch', scale=True)
     LOGGER.info(f"precomputed PCA: {score}")
     assert 0 <= score < 1e-6
 
 
-def test_pcr_batch_embedding(adata, embed_factory):
+def test_pcr_batch_embedding(adata):
     # use different embedding
     score = scIB.me.pcr_comparison(
         adata_pre=adata,
-        adata_post=embed_factory(adata, type_='full'),
+        adata_post=add_embed(adata, type_='full'),
```

Comment:
as this is now a function, does this need to be imported here? Or do you want to keep this as a fixture factory?

Reply:
The function is declared in `common.py` now, so it doesn't need to be imported explicitly for pytest to find it.